pennylane/tensorbox/numpy_box.py

```diff
@@ -0,0 +1,65 @@
+# Copyright 2018-2020 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This module contains the NumpyBox implementation of the TensorBox API.
+"""
+import numpy as np
+import pennylane as qml
+
+
+class NumpyBox(qml.TensorBox):
+    """Implements the :class:`~.TensorBox` API for ``numpy.ndarray``.
+
+    For more details, please refer to the :class:`~.TensorBox` documentation.
+    """
+
+    def __init__(self, tensor):
+        if not isinstance(tensor, np.ndarray):
+            tensor = np.asarray(tensor)
+
+        super().__init__(tensor)
+
+    def __array_ufunc__(self, ufunc, method, *inputs, **kwargs):

```

Comment:
Curious what this is...maybe add a docstring since it does not seem to be defined in a super class?

Reply:
done!

Reply:
I didn't spot the new docstring. Can you point out where it was added?

Reply:
I hadn't pushed it 🙈 I've now pushed it!